pingpong/models.py

```
@@ -1313,6 +1313,31 @@ class ConnectorConfig(Base):
 
     user_connectors = relationship("UserConnector", back_populates="connector_config")
 
+    @classmethod
+    async def list_all(cls, session: AsyncSession) -> list["ConnectorConfig"]:
+        stmt = select(ConnectorConfig)
+        result = await session.execute(stmt)
+        return [row.ConnectorConfig for row in result]
+
+    @classmethod
+    async def get_by_id(
+        cls, session: AsyncSession, id: int
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)